onvm/go.sh

```diff
@@ -52,6 +52,13 @@ then
     usage
 fi
 
+ports_detected=$("$RTE_SDK"/usertools/dpdk-devbind.py --status-dev net | sed '/Network devices using kernel driver/q' | grep -c "drv")
+if [[ $ports_detected -lt $ports ]]
```

Comment:
@rohit-mp @kevindweb I think this might be a bug since `$ports` represents the port mask while `$ports_detected` is the number of network devices using DPDK-compatible driver. Please correct me if I am wrong.

Reply:
Yes, you are right. Sorry for the mistake on my part. This bug is being tracked in issue #228 at the moment.